cheesy ass workaround for gcc-2:
i686-pc-linux-gnu-gcc -Wall -DLOCALEDIR=\"/usr/share/locale\" -march=i686 -O2 -pipe   -o pstree  pstree.o -lncurses 
fuser.c:103: warning: preprocessing directive not recognized within macro arg
fuser.c:103: warning: preprocessing directive not recognized within macro arg
fuser.c:103: warning: preprocessing directive not recognized within macro arg
fuser.c:103: warning: preprocessing directive not recognized within macro arg
fuser.c:103: warning: preprocessing directive not recognized within macro arg
fuser.c:103: warning: preprocessing directive not recognized within macro arg
fuser.c: In function `usage':
fuser.c:98: undefined or invalid # directive
fuser.c:101: undefined or invalid # directive
make[3]: *** [fuser.o] Error 1

--- src/fuser.c
+++ src/fuser.c
@@ -82,6 +82,11 @@ static void usage (const char *errormsg)
 	if (errormsg != NULL)
 		fprintf(stderr, "%s\n", errormsg);
 
+#ifdef WITH_IPV6
+# define _IPv6(x) x
+#else
+# define _IPv6(x)
+#endif
   fprintf (stderr, _(
     "Usage: fuser [ -a | -s | -c ] [ -n SPACE ] [ -SIGNAL ] [ -kimuv ] NAME...\n"
     "             [ - ] [ -n SPACE ] [ -SIGNAL ] [ -kimuv ] NAME...\n"
@@ -101,10 +106,10 @@ static void usage (const char *errormsg)
     "    -u        display user IDs\n"
     "    -v        verbose output\n"
     "    -V        display version information\n"
-#ifdef WITH_IPV6
+_IPv6(
     "    -4        search IPv4 sockets only\n"
     "    -6        search IPv6 sockets only\n"
-#endif
+)
     "    -         reset options\n\n"
     "  udp/tcp names: [local_port][,[rmt_host][,[rmt_port]]]\n\n"));
   exit (1);
